runbotics/runbotics-orchestrator-ui/src/components/Table/Table.styles.ts

```diff
@@ -23,32 +23,32 @@ export const DataTableWrapper = styled.div(({ theme }) => `
     box-shadow: ${theme.shadows[2]};
 `);
 
-const subRowStyles = css<{ isRowSelected?: boolean }>`
+const subRowStyles = css<{ isRowSelected?: boolean, isClickable?: boolean }>`
     background-color: ${(p) => (p.isRowSelected ? p.theme.palette.action.selected : p.theme.palette.grey[100])};
 
     &:hover {
-        cursor: pointer;
+        cursor: ${(p) => (p.isClickable ? 'pointer' : 'normal')};
         background-color: ${(p) => p.theme.palette.action.hover};
     }
 `;
 
-const rowStyles = css<{ isRowSelected?: boolean }>`
+const rowStyles = css<{ isRowSelected?: boolean, isClickable?: boolean }>`
     ${(p) => p.isRowSelected && `background-color: ${p.theme.palette.action.selected}`};
 
     &:hover {
-        cursor: pointer;
+        cursor: ${(p) => (p.isClickable ? 'pointer' : 'normal')};
```

Comment:
i think `props` or destructuring would be more consistent with the rest of the project